### PR TITLE
Added two missing desynth recipes

### PR DESCRIPTION
**Pull request by Noikar**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

Linen Slops and Cuffs desynth recipes added

### CODE REVIEW

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -4704,6 +4704,8 @@ INSERT INTO `synth_recipes` VALUES (75508,1,0,0,98,0,0,0,0,0,0,4100,4242,12420,0
 INSERT INTO `synth_recipes` VALUES (75509,1,0,0,103,0,0,0,0,0,0,4100,4242,12548,0,0,0,0,0,0,0,855,1225,654,655,1,6,2,2,'Adaman Cuirass (desynth)'); -- use synth lvl, wiki results
 INSERT INTO `synth_recipes` VALUES (75510,1,0,0,100,0,0,0,0,0,0,4100,4242,12932,0,0,0,0,0,0,0,855,1225,654,655,1,6,2,1,'Adaman Sabatons (desynth)'); -- use synth lvl, wiki results
 INSERT INTO `synth_recipes` VALUES (75511,0,0,0,0,0,0,0,0,0,101,4096,4238,615,4356,5568,5569,5688,0,0,0,6601,6601,6602,6602,6,8,6,8,'Omelette Sandwich'); -- BG Wiki and JP Wiki
+INSERT INTO synth_recipes VALUES(75512,1,0,0,0,0,23,0,0,0,0,4100,0,12729,0,0,0,0,0,0,0,818,819,807,807,2,4,1,2, "Linen Cuffs (desynth)");
+INSERT INTO synth_recipes VALUES(75513,1,0,0,0,0,25,0,0,0,0,4100,0,12857,0,0,0,0,0,0,0,818,818,819,819,2,3,6,7, "Linen Slops (desynth)");
```
Comment (TeoTwawki) [inline sql/synth_recipes.sql]: hey see how the rows right above yours cite their sources? also see those commented out rows for exactly what you are adding? https://github.com/LandSandBoat/server/blob/612595acd58f5c6b919fd4891bc1c87aae6dad1d/sql/synth_recipes.sql#L2661

also please see this issue I made and even pinned to the top of the issues page https://github.com/LandSandBoat/server/issues/885

edit:
I know I tend to be very blunt, my intention is to point these out not insult/be rude to anybody

**Review by TeoTwawki — CHANGES_REQUESTED** [review]

File: sql/synth_recipes.sql [hunk 2]
```diff
@@ -2658,8 +2658,8 @@ INSERT INTO `synth_recipes` VALUES (39906,1,0,0,0,0,12,0,0,0,0,4100,4242,12712,0
 -- INSERT INTO `synth_recipes` VALUES (39908,1,0,0,0,0,255,0,0,0,0,4100,4242,12737,0,0,0,0,0,0,0,834,816,820,823,1,4,2,1,'White Mitts (desynth)');
 INSERT INTO `synth_recipes` VALUES (39909,1,0,0,0,0,94,0,0,0,0,4100,4242,14301,0,0,0,0,0,0,0,851,816,816,821,1,3,6,1,'Errant Slops (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (39910,1,0,0,0,0,255,0,0,0,0,4100,4242,12969,0,0,0,0,0,0,0,818,819,819,819,9,1,1,1,'Cotton Kyahan (desynth)');
--- INSERT INTO `synth_recipes` VALUES (39911,1,0,0,0,0,255,0,0,0,0,4100,4242,12857,0,0,0,0,0,0,0,818,819,819,819,3,7,7,7,'Linen Slops (desynth)');
--- INSERT INTO `synth_recipes` VALUES (39912,1,0,0,0,0,255,0,0,0,0,4100,4242,14856,0,0,0,0,0,0,0,818,834,850,850,10,1,1,1,'Seer\'s Mitts (desynth)');
+INSERT INTO `synth_recipes` VALUES (39911,1,0,0,0,0,25,0,0,0,0,4100,0,12857,0,0,0,0,0,0,0,818,818,819,819,2,3,6,7,'Linen Slops (desynth)'); -- Wikia, BG has no info
+ --INSERT INTO `synth_recipes` VALUES (39912,1,0,0,0,0,255,0,0,0,0,4100,4242,14856,0,0,0,0,0,0,0,818,834,850,850,10,1,1,1,'Seer\'s Mitts (desynth)');
```
Comment (Xaver-DaRed) [inline sql/synth_recipes.sql]: Line 2662 - Revert this change.

File: sql/synth_recipes.sql [hunk 3]
```diff
@@ -2658,8 +2658,8 @@ INSERT INTO `synth_recipes` VALUES (39906,1,0,0,0,0,12,0,0,0,0,4100,4242,12712,0
 -- INSERT INTO `synth_recipes` VALUES (39908,1,0,0,0,0,255,0,0,0,0,4100,4242,12737,0,0,0,0,0,0,0,834,816,820,823,1,4,2,1,'White Mitts (desynth)');
 INSERT INTO `synth_recipes` VALUES (39909,1,0,0,0,0,94,0,0,0,0,4100,4242,14301,0,0,0,0,0,0,0,851,816,816,821,1,3,6,1,'Errant Slops (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (39910,1,0,0,0,0,255,0,0,0,0,4100,4242,12969,0,0,0,0,0,0,0,818,819,819,819,9,1,1,1,'Cotton Kyahan (desynth)');
--- INSERT INTO `synth_recipes` VALUES (39911,1,0,0,0,0,255,0,0,0,0,4100,4242,12857,0,0,0,0,0,0,0,818,819,819,819,3,7,7,7,'Linen Slops (desynth)');
--- INSERT INTO `synth_recipes` VALUES (39912,1,0,0,0,0,255,0,0,0,0,4100,4242,14856,0,0,0,0,0,0,0,818,834,850,850,10,1,1,1,'Seer\'s Mitts (desynth)');
+INSERT INTO `synth_recipes` VALUES (39911,1,0,0,0,0,25,0,0,0,0,4100,0,12857,0,0,0,0,0,0,0,818,818,819,819,2,3,6,7,'Linen Slops (desynth)'); -- Wikia, BG has no info
```
Comment (Xaver-DaRed) [inline sql/synth_recipes.sql:2661]: Line 2661 - Why remove the HQ crystal used?

File: sql/synth_recipes.sql [hunk 4]
```diff
@@ -2675,7 +2675,7 @@ INSERT INTO `synth_recipes` VALUES (39924,1,0,0,0,0,88,0,0,0,0,4100,4242,14253,0
 -- INSERT INTO `synth_recipes` VALUES (39925,1,0,0,0,0,255,0,0,0,0,4100,4242,14422,0,0,0,0,0,0,0,819,834,834,834,10,3,3,3,'Noct Doublet (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (39926,1,0,0,0,0,255,0,0,0,0,4100,4242,12601,0,0,0,0,0,0,0,818,819,819,819,7,6,6,6,'Linen Robe (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (39927,1,0,0,0,0,255,0,0,0,0,4100,4242,13950,0,0,0,0,0,0,0,823,855,844,844,1,2,1,1,'Elite Beret (desynth)');
--- INSERT INTO `synth_recipes` VALUES (39928,1,0,0,0,0,255,0,0,0,0,4100,4242,12729,0,0,0,0,0,0,0,818,819,807,807,3,4,2,2,'Linen Cuffs (desynth)');
+INSERT INTO `synth_recipes` VALUES (39928,1,0,0,0,0,23,0,0,0,0,4100,0,12729,0,0,0,0,0,0,0,818,819,807,807,2,4,1,2,;'Linen Cuffs (desynth)'); -- Wikia, bg wiki no info
```
Comment (Xaver-DaRed) [inline sql/synth_recipes.sql:2678]: Line 2678 - ,2,4,1,2,;'Linen Cuf- <-- Theres a badly placed character there. (the ;)
Also, why remove the HQ crystal?

**Review by Xaver-DaRed — CHANGES_REQUESTED** [review]

File: sql/synth_recipes.sql (hunk 2 above)
Comment (Noikar) [reply sql/synth_recipes.sql]: You want me to revert it to the wrong mats?

Comment (Xaver-DaRed) [reply sql/synth_recipes.sql]: Line 2662

Comment (Xaver-DaRed) [reply sql/synth_recipes.sql]: The change in said line unnecesarily removes a space between -- and the INSERT. That will make the CI complain

**Comment by Noikar** [comment]
Okay it's becoming too annoying to care about this PR anymore, someone else's problem now.
